EdFi.Ods.Utilities.Migration/logistics/scripts/modules/build-edfi-standard-descriptors.psm1

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = "Stop"
+$repositoryNames = @('EdFi.Ods.Utilities.Migration')
+& "$PSScriptRoot\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"  $repositoryNames 
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\tasks\TaskHelper.psm1')
```

Comment:
do we really need to use path resolving here? this is an independent repo

Reply:
I think we should be able to remove the path-resolver, TaskHelper, and packaging modules dependencies and not duplicate them here in this repo. If we also create a .nuspec file all this script is doing is copying the descriptors into the right folder structure for the nuget package, so it shouldn't be a lot of code.

Reply:
no . I removed it . Thanks

Reply:
yes  ,  I need it